### ISSUE TITLE
[REQUEST]OBDELEVEN

### ISSUE THREAD

**Issue by Fabio1968** [jilleb/mqb-pm#129]
Hi, I saw that on performance monitors there is some data that it takes from torque.  Would it be possible to have him read the data from obdeleven as well?  Thank you

**Comment by jilleb** [comment]
No, that wouldn't be possible. OBD11 does not share the data with other apps, there's no interface.
